library/Fission/User/Creator/Class.hs

```diff
@@ -84,6 +84,7 @@ instance
   create username pk email now =
     User
       { userPublicKey     = Just pk
+      , userExchangeKeys  = Just []
```

Comment:
should we pass exchange keys on user creation? Or create with an empty array and then flesh it out later?